### ISSUE TITLE
BUG: Current 2.0.1 master (57a524f) still breaks the Gutenberg Editor

### ISSUE THREAD

**Issue by kujain** [alleyinteractive/apple-news#677]
Continuing from the issue reported on #666 , this is still happening on the latest master:
```
SyntaxError: Unexpected token o in JSON at position 1
    at JSON.parse (<anonymous>)
    at t.value (https://insidehook.test/wp-content/plugins/apple-news-master/build/pluginSidebar.js?ver=2.0.2:1:32829)
    at finishClassComponent (https://insidehook.test/wp-includes/js/dist/vendor/react-dom.js?ver=16.8.4:14830:31)
    at updateClassComponent (https://insidehook.test/wp-includes/js/dist/vendor/react-dom.js?ver=16.8.4:14785:24)
    at beginWork (https://insidehook.test/wp-includes/js/dist/vendor/react-dom.js?ver=16.8.4:15733:16)
    at performUnitOfWork (https://insidehook.test/wp-includes/js/dist/vendor/react-dom.js?ver=16.8.4:19401:12)
    at workLoop (https://insidehook.test/wp-includes/js/dist/vendor/react-dom.js?ver=16.8.4:19441:24)
    at renderRoot (https://insidehook.test/wp-includes/js/dist/vendor/react-dom.js?ver=16.8.4:19524:7)
    at performWorkOnRoot (https://insidehook.test/wp-includes/js/dist/vendor/react-dom.js?ver=16.8.4:20431:7)
    at requestWork (https://insidehook.test/wp-includes/js/dist/vendor/react-dom.js?ver=16.8.4:20179:7)
```

I downloaded the current master (https://github.com/alleyinteractive/apple-news/commit/57a524f2cc4391c98f4b91b6b0ad2b9691a6b931) and on noticing there was no build/pluginSidebar.js included, I built it:
```
npm install
npm run-script build
```
This created the build folder with the pluginSidebar.js file (this was probably the reason why it was not showing up for me as I reported previously: https://github.com/alleyinteractive/apple-news/issues/666#issuecomment-529653884 ). 

Then on loading the Gutenberg enabled posts, the above error occurred.

**Comment by kevinfodness** [comment]
@kujain We just released version 2.0.2 which should fix issues for users in a few circumstances. If you look at the release channel, you'll see that there are `-built` tags for final code for a specific version including the built JS that is shipped to WordPress.org. So yes, you would need to build if you were using the latest `master`, but if you're downloading from WordPress.org or if you submodule the built tag you won't need to.

Let me know if v2.0.2 fixes problems you are experiencing!

**Comment by kevinfodness** [comment]
We also released v2.0.3 today to fix some JS errors that users were reporting. I'm going to close this issue, but please open a new one if you encounter any issues with v2.0.3. Thanks.
